apps/adk-runtime/adk_runtime/deps.py

```
@@ -0,0 +1,50 @@
+    url = os.environ.get("DATABASE_URL")
+    if not url:
+        return False
+    try:
+        with psycopg.connect(
+            url, options="-c search_path=agent_runtime", connect_timeout=2
+        ) as conn:
+            with conn.cursor() as cur:
+                cur.execute("SELECT 1")
+                cur.fetchone()
+        return True
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)